docs/basic/from_pg2.rst

```diff
@@ -4,10 +4,10 @@
 .. currentmodule:: psycopg
 
 
-Differences from ``psycopg2``
-=============================
+Differences to ``psycopg2``
```

Comment:
I don't think this is an improvement. _from_ seems more correct.

https://dictionary.cambridge.org/us/grammar/british-grammar/different-from-different-to-or-different-than

Reply:
Agree with Daniele. My reasoning is that these are the "Differences [encountered when moving] **from** psycopg2 [**to** psycopg3]"

Reply:
According to https://english.stackexchange.com/questions/36756/difference-to-or-difference-with-or-difference-from both "difference from" and "difference to" are wrong. I'll revert.